package.json

```diff
@@ -36,22 +44,30 @@
     "color",
     "colour",
     "finder",
+    "windows",
+    "explorer",
     "mavericks",
     "yosemite",
     "mac"
   ],
   "dependencies": {
     "chalk": "^1.1.1",
-    "finder-tag": "^1.0.2",
     "meow": "^3.6.0",
     "pify": "^2.3.0",
     "read-pkg-up": "^1.0.1"
   },
+  "optionalDependencies": {
```

Comment:
Pity npm doesn't give a way to specify OS-specific dependencies. I'm not sure `optionalDependencies` is what we want because they will install anyway but if they fail for a legitimate reason during installation then it will be ignored. What do you think? :-/